picasso/src/test/java/com/squareup/picasso/UtilsTest.java

```diff
@@ -116,4 +117,14 @@
     int id = Utils.getResourceId(resources, request);
     assertThat(id).isEqualTo(RESOURCE_ID_1);
   }
+
+  @Test @Config(reportSdk=9) public void useOkHttpByDefault() throws Exception {
+    Downloader downloader = Utils.createDefaultDownloader(Robolectric.application);
+    assertThat(downloader instanceof OkHttpDownloader).isTrue();
```

Comment:
Maybe change this to `assertThat(downloader).isInstanceOf(OkHttpDownloader.class)` ?